woocommerce/woocommerce-android
# Order Creation: add variable products

Closes: #5280 

This is based on @ThomazFB's PR #5499, please don't merge until it's merged.

### Description
This PR adds support for adding variable products to the order to create.

I reused the same adapter that the Variation's list screen uses, it differs from the screenshot on the ticket in the fact that it doesn't show the variation's price, IMO this should be OK for now until the designs are finalized, then we can add a new Adapter easily.

### Testing instructions
1. Make sure the order creation beta flag is enabled.
2. Open Order's list, and click on the FAB button.
3. Click on Add Product.
4. Click on a variable product.
5. Notice the list of the product's variations.
6. Confirm that only variations with prices are shown.
7. Select a variation.
8. Confirm that you are taken back to the order's form, and that the variation has been added.

### Images/gif
![Kapture 2021-12-29 at 17 58 54](https://user-images.githubusercontent.com/1657201/147686044-9c0d297f-71f9-4407-bd1d-af308648b72f.gif)
